.github/workflows/quay_binaries_push.yml

```diff
@@ -93,7 +56,7 @@ jobs:
           buildah manifest create oadp-cli-binaries
 
           buildah build \
-            --platform=linux/amd64,linux/arm64,linux/ppc64le,linux/s390x \
```

Comment:
Unless we won't be running this server on ppc or s390x, I would revert this change.
https://github.com/migtools/oadp-cli/blob/1c86f89e309e53402be70c41dc2ed4fd79b72ab1/Containerfile.download#L36

cc: @weshayutin